bcda/auth/plugin/alpha.go

```diff
@@ -193,7 +192,11 @@ func (p *AlphaAuthPlugin) RevokeAccessToken(token string) error {
 }
 
 func (p *AlphaAuthPlugin) ValidateAccessToken(token string) error {
-	return errors.New("not yet implemented")
+	t, err := p.DecodeAccessToken(token)
+	if err != nil {
+		return err
+	}
+	return t.Claims.(*CustomClaims).Valid()
```

Comment:
See auth/middleware.go for additional validations that should be incorporated here. Specifically:
* signed with the right algorithm in ParseToken()
* blacklisted check in RequireTokenAuth()
* aco string matches pattern and is in id in RequireTokenACOMatch() (this one should always be checked, not conditionally checked as it is now)

Reply:
the right algorithm is technically checked in `DecodeAccessToken()` which is the first line of this validation func